utils/api/configurations.js

```diff
@@ -70,8 +70,12 @@ const normalizeDescription = (text) => {
 }
 
 export const normalizeDate = (str) => {
-  const date = new Date(str)
-  return `${date.toDateString().substring(3)} at ${date.toLocaleTimeString()}`
+  if (str) {
```

Comment:
the value of `str` would be `null` if the "no deadline" checkbox had been selected.